tasty-hedgehog/src/Test/Tasty/Hedgehog.hs

```diff
@@ -0,0 +1,125 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
```

Comment:
Whoops - I didn't mean to include that in this branch, this PR was only meant to be about the Read instances. I'll clean that up shortly.